nautobot/extras/views.py

```diff
@@ -254,11 +506,88 @@ class ApprovalWorkflowStageInstanceResponseUIViewSet(
                 weight=100,
                 section=SectionChoices.LEFT_HALF,
                 fields="__all__",
+                value_transforms={
+                    "state": [helpers.render_approval_workflow_state],
+                },
             ),
         ],
     )
 
 
+class ApproverDashboardView(ObjectListViewMixin):
+    """
+    View for the dashboard of approval workflow stage instances waiting for the current user to approve.
```

Comment:
Feels like the dashboard view(s) should (eventually?) **not** have the standard `add/import` buttons that a standard ObjectListView has.